.github/workflows/cmake-script.yml

```diff
@@ -468,8 +468,9 @@ jobs:
             set (CTEST_DROP_SITE_INIT "my.cdash.org")
             # Change following line to submit to your CDash dashboard to a different CDash project
             #set (CTEST_DROP_LOCATION_INIT "/submit.php?project=HDF5")
-            #set (CMAKE_GENERATOR_TOOLSET "Intel C++ Compiler 2024,fortran=ifx")
+            set (CMAKE_GENERATOR_TOOLSET "Intel C++ Compiler 2024,fortran=ifx")
```

Comment:
Why is `fortran=ifx` necessary?
 
See `-DHDF5_BUILD_FORTRAN:BOOL=OFF` in line 476.

Reply:
In the next PR when I fix Intel compiling I will enable fortran